small-talk/app/chat/page.js

```diff
@@ -21,28 +21,31 @@ export default function Chat(){
             });
             setUser(receivedUsers);
         });
```

Comment:
I mentioned this in Jun's PR but it got merged. Are you able to adjust the formatting here with the indents?

Reply:
Seems like a preview error in github. We joined a chat to make sure that was the case.